px-dropdown-content.html

```diff
@@ -299,46 +337,61 @@
      * as well as closes the dropdown. Finally, it flips the opened flag.
      */
     _clickItem: function(evt) {
-
       // only handle the click if the item is not disabled
       if (!this.computedItems[evt.currentTarget.value].disabled) {
         /**
-        * Event fired when an element is clicked  on in the dropdown.
-        * @event px-dropdown-click
-        */
+         * Event fired when an element is clicked  on in the dropdown.
+         * @event px-dropdown-click
+         */
         this.fire('px-dropdown-click', evt);
 
         if (this.checkboxMode) {
-
           //try to toggle checkbox state
-          var checkbox = Polymer.dom(evt.target).querySelector('input');
+          var checkbox = Polymer.dom(evt.target).querySelector('input'),
+            items = [],
+            displayValues = '';
 
           //if we haven't found it it's probably because the click was actually
           //on the checkbox so just ignore
           if (checkbox) {
             checkbox.checked = !checkbox.checked;
             this._checkboxChanged(checkbox);
           }
+
+          this.items.forEach(function (item) {
```

Comment:
we try really hard not to use forEach because it's so much slower - can you please re-write this with a regular for loop?